openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1500,9 +1500,28 @@ Credential Refresh can be initiated by the Wallet independently from the Credent
 
 It is up to the Credential Issuer whether to update both the signature and the claim values, or only the signature.
 
-# Privacy Considerations
+# Privacy Considerations {#privacy-considerations}
 
-TBD
+Privacy harms can occur if information about a person is released to another party without the person's consent.
+True consent involves both the person knowing what information is being released and knowing who it is being released to, and for what purpose.
+The means to ensure the session's integrity to prevent disclosure to unintended parties, including wallets not under control of the End-User, are means to protect privacy.
+
+The Issuer needs to make choices about which claims are selectively disclosable (and should do so carefully).
```

Comment:
- change this to a more generic statement: `selective disclosure should be used to protect user privacy.`

discussed that there are credential format specific aspects - atomic credentials is another option - one claim per credential. and in anoncreds, all claims become selectively disclosable without the issuer choice. But agreed to not to talk about specific credential formats in VCI here.